src/web/browser/coreVitals/coreVitals.ts

```diff
@@ -33,11 +33,11 @@ export const coreVitals = (): void => {
 		name: string;
 		value: number;
 	};
-	
+
 	const nineDigitPrecision = (value: number) => {
 		// The math functions are to make sure the length of number is <= 9
 		return Math.round(value * 1_000_000) / 1_000_000;
-	}
+	};
```

Comment:
This is unrelated, but I was failing pre-push hooks without running prettier on this file

Reply:
I have a PR to fix it https://github.com/guardian/dotcom-rendering/pull/2995

Right now github actions are down so will merge on monday

Reply:
Ah okay nice 👍 I can rebase after that's merged